doc/index.rst

```diff
@@ -17,9 +17,9 @@ Installing the First Node
 -------------------------
 
 1. Install a recent, supported Linux distribution on a computer.
-2. Install docker. On Fedora or Centos::
-   
-   $ sudo dnf install docker 
+2. Install podman or docker. On Fedora or Centos::
+
+   $ sudo dnf install podman
```

Comment:
Hm. you guys are still sure you want to duplicate doc/cephadm/index.rst in doc/index.rst? Or should these changes go into cephadm/index.rst?

Reply:
I vote for Sebastian's plan.